dsb-gui/src/main/java/com/spectralogic/dsbrowser/gui/components/interruptedjobwindow/JobInfoPresenter.java

```diff
@@ -80,11 +84,12 @@
     private final JobWorkers jobWorkers;
     private final JobInterruptionStore jobInterruptionStore;
     private final LoggingService loggingService;
-    private final RecoverInterruptedJob.RecoverInterruptedJobFactory recoverInterruptedJobFactory;
     private final ButtonCell.ButtonCellFactory buttonCellFactory;
     private final DeepStorageBrowserPresenter deepStorageBrowserPresenter;
     private final DateTimeUtils dateTimeUtils;
-
+    private final SettingsStore settingsStore;
+    private final JobTaskElement jte;
```

Comment:
Can we call jet jobTaskElement?  I'm not going to remember what ate is when I see it next.